--- origsrc/goocanvas-0.11/configure.in	2008-09-09 14:31:46.000000000 -0500
+++ src/goocanvas-0.11/configure.in	2008-09-17 22:57:52.965372400 -0500
@@ -12,11 +12,11 @@
 # If interfaces changed but it is backwards compatible: AGE++, else AGE=0.
 #  i.e. the library supports interfaces LT_CURRENT - LT_AGE up to LT_CURRENT.
 #  if two libraries have identical CURRENT and AGE, then newest REVISION used.
-LT_CURRENT=4
-LT_AGE=1
-LT_REVISION=1
+CURRENT=4
+AGE=1
+REVISION=1
 
-LT_VERSION_INFO="$LT_CURRENT:$LT_REVISION:$LT_AGE"
+LT_VERSION_INFO="$CURRENT:$REVISION:$AGE"
 AC_SUBST(LT_VERSION_INFO)
 
 AM_MAINTAINER_MODE
